config/resolvers/201-clusterrolebinding.yaml

```diff
@@ -16,7 +16,6 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRoleBinding
 metadata:
   name: tekton-pipelines-resolvers
-  namespace: tekton-pipelines-resolvers
```

Comment:
Thanks for the PR @pst . I've been wondering why this shall not be in the reslover namespace? And I think`pipeline/config/201-rolebinding.yaml` shall be the correct file that gets rendered to release.yaml?

Reply:
ClusterroleBinding is not namespace scoped, as mentioned in https://github.com/tektoncd/pipeline/issues/5795